src/Controller/PageAdminController.php

```diff
@@ -113,7 +106,31 @@ public function treeAction(Request $request = null)
             'csrf_token' => $this->getCsrfToken('sonata.batch'),
         ]);
     }
+    
+    /**
+     * {@inheritdoc}
```

Comment:
Please do not add this comment. I know it is maybe on other bundles, but we are now trying to remove them all

Reply:
Updated.